test/provisioner/provision_ibmcloud_initializer.go

```diff
@@ -204,10 +203,20 @@ func initProperties(properties map[string]string) error {
 		if len(IBMCloudProps.CosServiceURL) <= 0 {
 			return errors.New("COS_SERVICE_URL was not set, example: s3.us.cloud-object-storage.appdomain.cloud")
 		}
+	} else {
```

Comment:
The `else` and `if` on the next line can be condensed into 1 line `else if`.

Reply:
Thanks, I have just made that change